### PR TITLE
Fix integration tests suite

### LINKED ISSUE #8: Fix integration test suite
Currently Travis CI builds are failing on every commit because of the following reasons:

- [x] Handshake unit test is failing
- [x] Autobahn's test server is outdated - should be updated to the latest `amphp/http-server` version
- [ ] Autobahn tests are failing

### PR DESCRIPTION
**Pull request by krlv**
Resolves #8

### CODE REVIEW

File: test/WebsocketTest.php [hunk 1]
```diff
@@ -252,7 +251,7 @@ public function testHandshake(Request $request, int $status, array $expectedHead
             $this->assertEquals($expectedHeaders, array_intersect_key($response->getHeaders(), $expectedHeaders));
 
             if ($status === Status::SWITCHING_PROTOCOLS) {
-                $this->assertNull(yield $response->getBody()->read());
+                $this->assertEmpty(yield $response->getBody()->read());
```
Comment (kelunik) [inline test/WebsocketTest.php]:
```suggestion
                $this->assertEmpty(yield ByteStream\buffer($response->getBody()));
```

File: composer.json [hunk 2]
```diff
@@ -38,7 +38,11 @@
         "league/climate": "^3",
         "league/uri-schemes": "^1.1",
         "phpunit/phpunit": "^6",
-        "infection/infection": "^0.7.1"
+        "infection/infection": "^0.7.1",
+        "amphp/artax": "^3.0.14",
+        "amphp/http-server-router": "^1.0.0",
+        "amphp/http-server-static-content": "dev-master",
```
Comment (kelunik) [inline composer.json]:
```suggestion
        "amphp/http-server-static-content": "^1",
```

**Review by kelunik — COMMENTED** [review]
Two smaller nits, otherwise fine.

**Comment by krlv** [comment]
Thanks for the tip! 👍 Fixed.

**Pull request merged**